packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -205,18 +252,20 @@ export class TableRow<
 
     private dataRecordChanged(): void {
         this.updateCellStates();
+        this.updateCellIndentLevels();
```

Comment:
Is this needed? `updateCellIndentLevels` doesn't use `dataRecord`.

Reply:
Nope, I think you're right. Removed.